backend/project_review_coordination/activity_log_creators.py

```diff
@@ -1,11 +1,11 @@
 from activity_log.models import LogEntry, EventType
 
 
-BUNDLE_CLAIMED = "BUNDLE_CLAIMED"
-BUNDLE_UNCLAIMED = "BUNDLE_UNCLAIMED"
-TIME_ADDED = "BUNDLE_TIME_ADDED"
-BUNDLE_EXPIRED = "BUNDLE_EXPIRED"
-BUNDLE_COMPLETED = "BUNDLE_COMPLETED"
+BUNDLE_CLAIMED = "PROJECT_REVIEW_BUNDLE_CLAIMED"
+BUNDLE_UNCLAIMED = "PROJECT_REVIEW_BUNDLE_UNCLAIMED"
+TIME_ADDED = "PROJECT_REVIEW_BUNDLE_TIME_ADDED"
```

Comment:
I think Sheena was referring to "TIME_ADDED" not starting with "BUNDLE_" with her inconsistency comment.